feathernotes/data/feathernotes.desktop

```diff
@@ -2,7 +2,8 @@
 Version=1.0
 Name=FeatherNotes
 Comment=Lightweight notes manager
-Comment[hu]=Könnyed jegyzetkezelő
```

Comment:
The above line is removed by mistake; isn't it?

Reply:
Sorry, fixed.

Reply:
Thanks!